my.css

```diff
@@ -0,0 +1,236 @@
+.header .header {
+	background: black;
+	position: relative;
+}
+
+.logo {
+	position: absolute;
+	width: 98px;
+	height: 37px;
+	left: 52px;
+	top: 36px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)